### PR TITLE
add answer_content to question document

### PR DESCRIPTION
**Pull request by LeoMcA**

### CODE REVIEW

File: docs/architecture/decisions/0003-es-aaq-documents.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# 3. AAQ structure in Search v2
+
+Date: 2020-10-27
+
+## Status
+
+Accepted
+
+## Context
```
Comment (akatsoulas) [inline docs/architecture/decisions/0003-es-aaq-documents.md:9]: I would also add something about the architecture itself which is different from what exists now (denormalization)

File: kitsune/search/v2/documents.py [hunk 2]
```diff
@@ -93,7 +93,17 @@ def get_model(cls):
...
 class QuestionDocument(SumoDocument):
-    """"""
+    """
+    ES document for Questions. Every Question in DB gets a QuestionDocument in ES.
+
+    Parent class to AnswerDocument, so most fields are prefixed with "question_".
+    QuestionDocument and AnswerDocument are stored in the same index, so QuestionDocument
+    cannot define a field with the same name as AnswerDocument.
+    Distinguish QuestionDocuments from AnswerDocuments by filtering on whether the
+    document has a null "created" field.
```
Comment (akatsoulas) [inline kitsune/search/v2/documents.py:103]: I wouldn't add how to distinguish between the two. I would talk more about the arch - denormalized version of the AAQ for ES and about the fact that `QuestionDocument` is the parent class of AnswerDocument but in the end it's a single document.

File: kitsune/search/v2/documents.py [hunk 3]
```diff
@@ -159,7 +175,18 @@ def get_queryset(cls):
...
 class AnswerDocument(QuestionDocument):
-    """"""
+    """
+    ES document for Answers. Every Answer in DB gets an AnswerDocument in ES.
+
+    Child class to QuestionDocument and stored in the same index, so AnswerDocument
+    cannot define a field with the same name as one in QuestionDocument.
+    Most QuestionDocument fields are prefixed by "question_".
+    Distingish AnswerDocuments from QuestionDocuments by filtering on whether the document
+    has a non-null "created" field.
```
Comment (akatsoulas) [inline kitsune/search/v2/documents.py:185]: Similar to the comment above. Let's mostly focus on the arch here and the class itself. Not on how to use it

**Review by akatsoulas — APPROVED** [review]
We can optimize the documentation later. This is a good first step

File: docs/architecture/decisions/0003-es-aaq-documents.md (hunk 1 above)
Comment (LeoMcA) [reply docs/architecture/decisions/0003-es-aaq-documents.md:9]: Do we use nested/joins currently? I'm not sure we do, we just needed to be cautious not to introduce it while re-modelling our data. I'll add a note about that.

**Pull request merged**